sources/api/shibaken/README.md

```diff
@@ -11,6 +11,8 @@ shibaken is used to fetch data from the instance metadata service (IMDS) in AWS.
 shibaken can:
 * Fetch and populate the admin container's user-data with authorized ssh keys from the IMDS.
 * Perform boolean queries about the AWS partition in which the host is located.
+* Support Bottlerocket refraining from joining a Kubernetes cluster if it is warming up in an autoscaling
```

Comment:
This needs to be phrased in terms of a general feature - "wait until the instance is marked as `InService` before continuing" or something.